products/mozilla_org/test_mozilla_org.md

```diff
@@ -0,0 +1,66 @@
+* Terminology is consistent with product, and between web pages
+* Brand names remain in English
+* Product names comply to Mozilla guideline and adhere to what the community has agreed to
+* No corrupted characters
+* Click on the links on the page, which should take you to the pages of the same language if they are localized, or they will be redirected to en-US if the pages are not.  
+* Nav bar terms consistent with the page titles they are linked to (except when Nav bar term is shortened due to space limitation)
+* Footer links don’t overlap with one another
+
+You can make linguistic changes directly in [Pontoon](https://pontoon.mozilla.org/projects/mozillaorg/), [Locamotion](https://mozilla.locamotion.org/projects/mozilla_lang/) or [GitHub](https://github.com/mozilla-l10n/www.mozilla.org/).  
+
+### Functionality Testing
+* Click the download link, you should be able to download the product in your language, if it is localized, such as Firefox
+* Font support and readability
+* Footer: verify that the translation of the link is coherent and the link is functional.
+* Language list: Is your language listed as one of the options? Check https://www-dev.allizom.org/en-US/ to confirm.
+* Deliberately type a wrong broken link, verify whether [404 page](https://www-dev.allizom.org/404/) is in the same language
+* In RTL (right to left languages), both target language and English flows in the correct directions.  The mirroring effect is correct as defined by Mozilla UX team. Indentation and alignment are correct
+
+
+### Testing for Fundraising Campaigns: 
+The annual fundraise campaign is localized in many languages, but not all languages. 
+* Date and time format is country specific
+* Currency availability
+* Postal/zip code in the proper format
+* Mailing address format appropriate for the country
+* Payment system available for the targeted country
+* Country list in your language and order
+
+### Compatibility Testing:
+* Test the page layout in other major browsers and on other platform
+* Test the page layout on the leading home grown browser if available
+* Test the page layout on mobile devices of major platforms
+
+## When Can I See the Localized Page on the Production Server? 
+
+Code push to the production server is made almost daily.  When a brand new page is being localized, the page won't go live until it is fully localized for the very first time. Once a page is on the production server, an ongoing maintenance is required, as updates take place on a irregular basis. New strings added to existing pages are visible in the production server only when localized, unless they're in a shared file.  If localized content is not kept up, either the page is completely out of date over time with old designs and message, or the site contains mixture of English and target language, we may have to disable it if the latter is the case. Check the team [web dashboard](https://l10n.mozilla-community.org/webdashboard/) for updates.  
```

Comment:
We need to simplify this, e.g.

```
Updated translations are pushed to the production server almost daily. 

When a brand new page is available for localization, it won't be enabled in production until it's fully localized. When existing pages receive updates with new strings, this new content won't be displayed on production until localized, to avoid displaying a mix of English and localized text. 
In some cases pages receive major updates that require a complete rewrite of the template: if this happens, the old template is kept online only for a short period of time and, when removed, it will cause the URL to redirect users to the English version.
```